scripts/sync-project-board.sh

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Vendored from kubernetes-sigs/sig-windows-tools/hack/sync-project-board.sh
+#  * commit: 09de175041779f35595dee339c2463b2eb194555
+#  * link: https://github.com/kubernetes-sigs/sig-windows-tools/blob/09de175041779f35595dee339c2463b2eb194555/hack/sync-project-board.sh
+
+
+set -e
+set -u
+set -o pipefail
+
+## DESCRIPTION:
+##
+## This script queries all repos in a given github org and adds the issues 
+## with label 'sig/auth' to a specified project board.
+##
+## REREQS:
+##
+## This script assumes there is a github PAT in the GITHUB_TOKEN env var
```

Comment:
Whose PAT does this use?

Reply:
We'll need to generate a PAT and use it. I've been testing it with my own PAT here: https://github.com/aramase/sig-auth-tools/actions/runs/3853956718/jobs/6567449739. We'll need to set the PAT as a secret with name `SYNC_PROJECTS_PAT` before the action is run.

Reply:
Are we able to use one of the new style access token with the fine grained authz?

Reply:
Not at the moment! fine-grained PAT doesn't support GraphQL API (xref: https://github.blog/2022-10-18-introducing-fine-grained-personal-access-tokens-for-github/#coming-next)